dashboard/app/views/admin_standards/import_standards.html.haml

```diff
@@ -0,0 +1,57 @@
+:ruby
+  courses = [
+    ["", ""],
+    ["Course A 2019", "coursea-2019"],
+    ["Course B 2019", "courseb-2019"],
+    ["Course C 2019", "coursec-2019"],
+    ["Course D 2019", "coursed-2019"],
+    ["Course E 2019", "coursee-2019"],
+    ["Course F 2019", "coursef-2019"],
+    ["Pre-Express 2019", "pre-express-2019"],
+    ["Express 2019", "express-2019"]
+  ]
```

Comment:
I don't love that I used a hardcoded list here, means one more thing to track with versioning. We're only interested in standards associations for 2019 and beyond and only for CSF and this happens so rarely, it might not be too terrible? Is there a better way to do it that still lets us import by course?

Reply:
You could produce a future-proofed list by searching for scripts with curriculum_umbrella == 'CSF' and version_year >= '2019'